app/src/main/java/org/schabi/newpipe/player/event/PlayerGestureListener.java

```diff
@@ -69,26 +69,22 @@ public void onSingleTap(@NotNull final MainPlayer.PlayerType playerType) {
         if (DEBUG) {
             Log.d(TAG, "onSingleTap called with playerType = [" + player.getPlayerType() + "]");
         }
-        if (playerType == MainPlayer.PlayerType.POPUP) {
 
-            if (player.isControlsVisible()) {
-                player.hideControls(100, 100);
-            } else {
-                player.getPlayPauseButton().requestFocus();
```

Comment:
I can only guess (or run git blame), but I think requesting focus here makes sense when you have a device with a physical keyboard which allows you to pause / play the video by pressing the spacebar. Putting the focus on the play/pause button is only reasonable when using the popup player (the main player has more buttons, so switching the focus might be weird for the user, especially when using NewPipe on an Android TV with a DPad). I'd keep that line and add a little hint.

Reply:
Uhm, sorry if I wrote that in a way leaving room for misinterpretation. The pop-up player does not work on TVs, because it cannot be moved (and that makes the whole feature useless...). The focus request is for phones with physical keyboards.

Reply:
> The focus request is for phones with physical keyboards.

This is such a niche that I don't think it's even worth implementing.
I also don't know if it's expected that the playPause-Button gains focus when e.g. the player is paused via headphone.
Particularly if the popup player will be replace by a PiP.

More insight:
[There are currently like 3 phones sold that have a physical keyboard](https://www.androidauthority.com/keyboard-phones-845839/) and all also have a touchscreen...

Reply:
Removed commit.